### ISSUE TITLE
Add an easy way to track major electives that need to be updated every semester

### ISSUE THREAD

**Issue by churichard** [TigerAppsOrg/TigerPath#209]
Some majors, like WWS and ANT, update their electives requirements every semester. Perhaps we can have these majors reference a different electives file so that it's easier to update them.

**Comment by bnehoran** [comment]
Yeah, I agree. We should have a way to separate the major structure from the course lists if desired, so that then we only need to maintain the course lists.
